itsUk45/Property-Pro-Lite-
# #167271196 update user interface

What does this PR do?

    update user interface modules

Description of Task to be completed

-update ui.
-improve page navigation.
- add index and details pages

How should this be manually tested?

    clone the repo and switch to the branch ch-ui-update2354662
    -run nodemon app to start the server
    -run npm run coveralls to see the test output

Any background context you want to provide?
-N/A

What are the relevant pivotal tracker stories?
   - #167271196
    Screenshots (if appropriate)
    -N/A
